widgetssdk/src/main/java/com/glia/widgets/di/GliaCoreImpl.kt

```diff
@@ -86,19 +86,15 @@ internal class GliaCoreImpl : GliaCore {
         Glia.getQueues(requestCallback)
     }
 
-    override fun queueForEngagement(queueId: String, visitorContextAssetId: String?, callback: Consumer<GliaException?>) {
-        Glia.queueForEngagement(queueId, visitorContextAssetId, callback)
-    }
-
     override fun queueForEngagement(
-        queueId: String,
+        queueIds: Array<String>,
```

Comment:
Maybe this should have been written in core sdk PR, but anyways, why we're using `Array` instead of more flexible and recommended by Kotlin `List`

Reply:
But you approved it https://github.com/salemove/android-sdk/pull/681

Reply:
Yes, but that was `Java` code.
This is not critical, but I would prefer changing such things to Kotlin's recommended way during migration.

Reply:
Let's discuss and implement this in a separate task.